src/metalperformanceshaders.cs

```diff
@@ -5088,6 +5323,151 @@ interface MPSCnnUpsamplingBilinearGradientNode {
 		double ScaleFactorY { get; }
 	}
 
+	interface IMPSNNLossCallback { }
+
+	[Protocol]
+	[BaseType (typeof(NSObject))]
+	[Model (AutoGeneratedName = true)]
```

Comment:
why a model here ? it's not a `*Delegate` and we can't provide a directly usable implementation

Reply:
Good catch